.github/workflows/validate-spec.sh

```diff
@@ -16,9 +16,15 @@ do
 
   # Ensure spec can be parsed
   name=$(rpmspec --srpm  --define "with_check 0" --qf "%{NAME}" -q $spec 2>/dev/null )
-  if [[ -z $name ]] 
+  if [[ -z $name ]]
   then
-    echo "Not able to parse $spec, skipping"
+    echo "Not able to parse $spec, skipping" >> bad_specs.txt
+    continue
+  fi
+  name=$(rpmspec --srpm  --define "with_check 1" --qf "%{NAME}" -q $spec 2>/dev/null )
+  if [[ -z $name ]]
+  then
+    echo "Not able to parse $spec, skipping" >> bad_specs.txt
```

Comment:
Let's also mention which `--define` were was used to make debugging easier.

Reply:
Done